tests/platform_tests/test_platform_info.py

```diff
@@ -150,7 +150,7 @@ def check_vendor_specific_psustatus(dut, psu_status_line):
         if "201811" in dut.os_version or "201911" in dut.os_version:
             psu_line_pattern = re.compile(r"PSU\s+(\d)+\s+(OK|NOT OK|NOT PRESENT)")
         else:
-            psu_line_pattern = re.compile(r"PSU\s+(\d+)\s+\w+\s+\w+\s+\w+\s+\w+\s+\w+\s+(OK|NOT OK|NOT PRESENT)\s+(green|amber|red|off)")
+            psu_line_pattern = re.compile(r"PSU\s+(\d+).*?(OK|NOT OK|NOT PRESENT)\s+(green|amber|red|off)")
```

Comment:
This regex becomes very lenient. It could miss issues with column alignment.

Reply:
curious, the output is in json format, why use regex. Is modifying the output an option to better suit to our testing needs.

Reply:
Using regex to parse the output is error prone. Suggest to use method `SonicHost::show_and_parse` to parse the tabulate output. It would be more robust.

Reply:
changed to show_and_parse method